modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/MockedStructuresTest.java

```diff
@@ -275,6 +285,58 @@ private void mockMetastore() throws Exception {
         when(cursorMocked.iterator()).thenReturn(itMock);
     }
 
+    @Test
```

Comment:
does it make sense to verify that options properly passed to the dzManager? I'm mean if we put `replicas=4`, it will  causes an invocation of dzManager with proper config

Reply:
Thanks, the tests have been improved by checking the configuration changes.